network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/tck/GroundTest.java

```diff
@@ -6,32 +6,23 @@
  */
 package com.powsybl.network.store.iidm.impl.tck;
 
-import static org.junit.Assert.assertNull;
-
 import org.junit.jupiter.api.Test;
 
-import com.powsybl.iidm.network.Network;
 import com.powsybl.iidm.network.tck.AbstractGroundTest;
-import com.powsybl.iidm.network.test.TwoVoltageLevelNetworkFactory;
 
 /**
  * @author Ayoub LABIDI <ayoub.labidi at rte-france.com>
  */
 class GroundTest extends AbstractGroundTest {
+
     @Test
     void test() {
-        // FIXME
+        // FIXME remove this test when we find why NetworkObjectIndex.getGrounds(String voltageLevelId) doesn't always return all expected result
+        // (some results are missing from groundCache)
```

Comment:
Ground equipments are missing in different NetworkStoreClient implementions (BufferedNetworkStoreClient, CachedNetworkStoreClient, PreloadingNetworkStoreClient) ...
Ground is also missing in VoltageLevelImpl.getConnectables() method ...